lib/edda-server/src/change_set_processor_task.rs

```diff
@@ -148,22 +150,6 @@ impl ChangeSetProcessorTask {
             .filter_map(|maybe_elapsed_item| maybe_elapsed_item.ok());
 
         let app = ServiceBuilder::new()
-            .layer(
-                MatchedSubjectLayer::new()
-                    .for_subject(EddaRequestsForSubject::with_prefix(prefix.as_deref())),
-            )
-            .layer(
-                TraceLayer::new()
-                    .make_span_with(
-                        telemetry_nats::NatsMakeSpan::builder(connection_metadata).build(),
-                    )
-                    .on_response(telemetry_nats::NatsOnResponse::new()),
-            )
-            .layer(
-                PostProcessLayer::new()
-                    .on_success(DeleteMessageOnSuccess::new(stream.clone()))
-                    .on_failure(DeleteMessageOnFailure::new(stream)),
-            )
```

Comment:
If you compress the message and double ack it, none of the NATS stuff is 1) necessary or 2) will work... telemetry is an unknown here as well.